lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -904,12 +920,19 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
         # @option opts [String, Array<String>] :dev_builds_repos The repo(s)
         #                                       to check for dev builds in.
         #
+        # @note puppet-agent requires :dev_builds_repos to be set to the
+        #   puppet collection(s) to set the build server URL appropriately.
+        #
         # @note This method only works on redhat-like and debian-like hosts.
         #
         def install_puppetlabs_dev_repo ( host, package_name, build_version,
                                   repo_configs_dir = 'tmp/repo_configs',
                                   opts = options )
           variant, version, arch, codename = host['platform'].to_array
+          if variant !~ /^(fedora|el|centos|debian|ubuntu|cumulus)$/
```

Comment:
Okay. Tbh I'm fuzzy on what the logic should be for the PE-only agent platforms (like `sles`), so I was asking out of curiosity at this point.